data-stories/upper-valley-land-trust/uvlt analyses abigail.ipynb

```diff
@@ -842,22 +842,22 @@
        "13                  Emerson        Norwich    VT   05055      24   \n",
        "14                Killebrew        Hanover    NH   03755      16   \n",
        "\n",
-       "                  Town  LandOwnerTownID DeceasedDate    ...      E-2010-11  \\\n",
```

Comment:
@jayxzhou did you mean to modify abigail's notebook?  Please confirm, or if this was a mistake please revert back to the original version of this file